gen/typescript/qdrant/cloud/cluster/v1/cluster_pb.d.ts

```diff
@@ -2106,6 +2174,48 @@ export enum ClusterPhase {
  */
 export declare const ClusterPhaseSchema: GenEnum<ClusterPhase>;
 
+/**
+ * ClusterNodeState represents the state of a cluster node.
+ *
+ * @generated from enum qdrant.cloud.cluster.v1.ClusterNodeState
+ */
+export enum ClusterNodeState {
+  /**
+   * The state is unspecified.
+   *
+   * @generated from enum value: CLUSTER_NODE_STATE_UNSPECIFIED = 0;
+   */
+  UNSPECIFIED = 0,
+
+  /**
+   * The node is starting.
+   *
+   * @generated from enum value: CLUSTER_NODE_STATE_STARTING = 1;
+   */
+  STARTING = 1,
+
+  /**
+   * The node is healthy.
+   *
+   * @generated from enum value: CLUSTER_NODE_STATE_HEALTHY = 2;
+   */
+  HEALTHY = 2,
+
+  /**
+   * The node is unhealthy.
+   *
+   * TODO Do we need this: CLUSTER_NODE_STATE_SUSPENDED = 4;
```

Comment:
Yes, this makes sense: This way we can show that the pods are not running, but the PVCs are still there.

Reply:
Added the state